tools/scripts/generate_binary_build_matrix.py

```diff
@@ -205,21 +205,20 @@ def get_wheel_install_command(os: str, channel: str, gpu_arch_type: str, gpu_arc
         index_arg = "--index-url" if channel == "nightly" else "--extra-index-url"
         return f"{whl_install_command} {index_arg} {get_base_download_url_for_repo('whl', channel, gpu_arch_type, desired_cuda)}"
 
-def generate_conda_matrix(os: str, channel: str, with_cuda: str) -> List[Dict[str, str]]:
+def generate_conda_matrix(os: str, channel: str, with_cuda: str, limit_win_builds: str) -> List[Dict[str, str]]:
     ret: List[Dict[str, str]] = []
     arches = ["cpu"]
     python_versions = PYTHON_ARCHES_DICT[channel]
 
-    if with_cuda == ENABLE:
-        if os == "linux":
-            arches += mod.CUDA_ARCHES
-        elif os == "windows":
-            # We don't build CUDA 10.2 for window see https://github.com/pytorch/pytorch/issues/65648
-            arches += list_without(mod.CUDA_ARCHES, ["10.2"])
+    if with_cuda == ENABLE and os == "linux":
+        arches += mod.CUDA_ARCHES
 
     if os == "macos-arm64":
         python_versions = list_without(python_versions, ["3.7"])
 
+    if os == "windows" and limit_win_builds == ENABLE:
+        python_versions = [ python_versions[0] ]
```

Comment:
this would be varying across the channels, e.g. release could be testing just 3.7, while nightly would be testing 3.8.

Reply:
Yes, we hopefully move release to 3.8 really soon :)